pywincffi/kernel32/overlapped.py

```diff
@@ -0,0 +1,71 @@
+"""
+Overlapped
+----------
+
+A module containing Windows functions for working with OVERLAPPED objects.
+"""
+
+from pywincffi.core import dist
+from pywincffi.core.checks import NON_ZERO, input_check, error_check
+from pywincffi.exceptions import WindowsAPIError
+from pywincffi.wintypes import HANDLE, OVERLAPPED, wintype_to_cdata
+
+
+def GetOverlappedResult(hFile, lpOverlapped, bWait):
+    """
+    Retrieves the results of an overlapped operation on the specified file,
+    named pipe, or communications device. To specify a timeout interval or
+    wait on an alertable thread, use GetOverlappedResultEx.
+
+    .. seealso::
+
+        https://msdn.microsoft.com/en-us/library/ms683209
+
+    :param HANDLE hFile:
```

Comment:
Need the path to the class here so it can be linked properly in the documentation.